delivery_free_fee_removal/models/sale_order_line.py

```diff
@@ -19,6 +19,6 @@ def _compute_is_free_delivery(self):
     @api.depends("is_free_delivery")
     def _get_to_invoice_qty(self):
         free_delivery_lines = self.filtered("is_free_delivery")
-        free_delivery_lines.qty_to_invoice = 0
+        # free_delivery_lines.qty_to_invoice = 0
         other_lines = self - free_delivery_lines
         super(SaleOrderLine, other_lines)._get_to_invoice_qty()
```

Comment:
you can actually remove the whole override of `_get_to_invoice_qty`: the only thing it was doing was to set `qty_to_invoice` to 0